cfme/common/__init__.py

```diff
@@ -238,16 +241,37 @@ def step(self, *args, **kwargs):
         self.prerequisite_view.toolbar.policy.item_select('Manage Policies')
 
 
+class AssignedTags(ParametrizedView):
+    PARAMETERS = ("tag",)
+    ALL_TAGS = ".//a[contains(@class, 'pf-remove-button')]"
+    tag = Text(ParametrizedLocator(
+        ".//div[@class='category-label']/parent::li/following-sibling::"
+        "li/descendant::a[contains(@class, 'pf-remove-button')]")
+    )
+
+    def remove(self):
+        self.tag.click()
+
+
 class TagPageView(BaseLoggedInPage):
     """Class represents common tag page in CFME UI"""
     title = Text('#explorer_title_text')
     table_title = Text('//div[@id="tab_div"]/h3')
 
     @View.nested
     class form(View):  # noqa
-        tags = Table("//div[@id='assignments_div']//table")
-        tag_category = BootstrapSelect(id='tag_cat')
-        tag_name = BootstrapSelect(id='tag_add')
+        tags = VersionPicker({
+            Version.lowest(): Table("//div[@id='assignments_div']//table"),
+            "5.11": ParametrizedView.nested(AssignedTags)
```

Comment:
I'm confused by this implementation, is there any reason tags can not also be a `@View.nested` object. Unless I'm missing something, I have to do `view.form.tags.view_class.tag` to get the tag? 

I would expect to be able to do `view.form.tags` to get all the tags assigned to the VM.

Reply:
ok, so...
I used `ParametrizedView` here, because it's easy to use with `VersionPicker` and pick one specific tag. The example is also in this PR
```
view = view.form.tags(category)
view.remove()
```
The code above removes the tag and this is used in teardown.

I'd say, the way you expect it to work should be possible as well, but for now I'd prefer to keep it like that.

To get all tags, `get_tags` method is used and it uses information from Details page
![image](https://user-images.githubusercontent.com/42433123/59763597-35fe7c00-929a-11e9-9925-9ba24f3df31e.png)

https://github.com/ManageIQ/integration_tests/blob/master/cfme/common/__init__.py#L427

Reply:
@lina-nikiforova Thanks for the explanation! Could you provide a `docblock` in assigned tags to explain how these should be used. Additionally, is it possible to read the tag? I am using the following code block.

```python,
view = navigate_to(vm, "EditTags")
view = view.form.tags("Workload")
view.tag.read()
Out[25]: u'Remove'
```
![Screenshot from 2019-06-19 10-31-01](https://user-images.githubusercontent.com/44065123/59774437-71d71880-927d-11e9-9ae8-696028180bd2.png)

Reply:
@john-dupuy thanks, John. I implemented the read function and added the docblock.